src/libexpr/flake/flake.cc

```diff
@@ -373,6 +373,29 @@ LockedFlake lockFlake(
         {
             debug("computing lock file node '%s'", printInputPath(inputPathPrefix));
 
+            auto overrides2 = overrides;
+            for (auto & [inputPath, inputOverride] : overrides2) {
+                auto inputPath2(inputPath);
```

Comment:
It feels like you're doing some useless copies and move+copy. Could this be replaced by:

```suggestion
            for (auto [inputPath, inputOverride] : overrides) {
```

(and replacing `inputPath2` by `inputPath` below)

?

Reply:
I guess so, not too experienced with C++, so I'll just try it out :D